ui/admin/app/templates/scopes/scope/aliases/alias/index.hbs

```diff
@@ -23,12 +23,76 @@
       </nav.link>
     </Rose::Nav::Tabs>
   </page.navigation>
-
+  {{#if (can 'delete model' @model)}}
+    <page.actions>
+      <Hds::Dropdown as |dd|>
+        <dd.ToggleButton
+          data-test-manage-dropdown
+          @text={{t 'actions.manage'}}
+          @color='secondary'
+        />
+        <dd.Interactive
+          data-test-manage-alias-dropdown-delete
```

Comment:
None blocking comment: are we planning on using these `data-test-*` selectors for tests?

Reply:
thanks, we don't need this, removed